data/856/679/73/85667973.geojson

```diff
@@ -34,6 +34,9 @@
     "name:eng_x_preferred":[
         "Neutral Zone"
     ],
+    "name:fra_x_preferred":[
+        "Neutral Zone"
```

Comment:
Nit: Really?

Reply:
Hmm interesting.. I'll double check this one, but will likely import as-is with a potential follow up if that works..

Reply:
That works for me